src/components/index.tsx

```diff
@@ -19,8 +19,10 @@ import * as React from "react"
 import * as ReactDOM from "react-dom"
 import * as Relay from "react-relay"
 
+import { configureUrlQuery } from "react-url-query"
 import { artsyNetworkLayer } from "../relay/config"
 import * as Artsy from "./artsy"
+import history from "./history"
```

Comment:
Right right, but it’s not entirely clear to me _why_ you need access to the history?

In any case, loading the module at the file level and the way the module is defined means it’s always going to be making that history object. Is there some way to only create the history object _when a component really needs it_ (lazily) and _only_ in the browser?